apps/src/netsim/NetSimRouterNode.js

```diff
@@ -1418,7 +1407,7 @@ NetSimRouterNode.prototype.getCurrentDataRate = function () {
 /**
  * When the message table changes, we might have a new message to handle.
  * Check for and handle unhandled messages.
- * @param {messageRow[]} rows
+ * @param {message[]} rows
```

Comment:
This doc change looks like a mistake; I think `onMessageTableChange_` still takes `messageRow[]`.